### DISCUSSION TITLE
Can't merge PRs due to test failures in the `OptimizationApplication`

### DISCUSSION THREAD

**Discussion by avdg81** [KratosMultiphysics/Kratos#11766] [General]
Hi all,
Since a few days we note that we frequently can't merge PRs due to test failures in the `OptimizationApplication`. The attached screenshot shows an example of such a test failure. As far as we know, these test failures are _unrelated_ to the changes that we try to merge. So I would like to know whether there are more contributors who also suffer from these test failures. If there are, I would like to kindly ask the maintainer(s) of the `OptimizationApplication` to have a look into this. Many thanks in advance for your response.

![test_failure](https://github.com/KratosMultiphysics/Kratos/assets/28814468/b10230f7-e847-4038-9d3a-fc2ceb401cea)

**Comment by rubenzorrilla** [comment]
Hi @avdg81. We're already taking care of it (see #11763). In any case, thanks for reporting.

**Reply by avdg81** [reply]
Great to see that you are already taking care of this. Many thanks!

**Comment by sunethwarna** [comment]
Could you try merging the master now? We pushed a fix from #11760.

**Reply by avdg81** [reply]
Thanks for your swift reply! I have rebased one of the branches that we attempted to merge. Hopefully the build pipelines will now succeed.

**Comment by avdg81** [comment]
The issue has been solved.
